ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/DB/AcidState.hs

```diff
@@ -117,6 +118,11 @@ instance Buildable NewPendingError where
     build (NewPendingFailed npf) =
         bprint ("NewPendingFailed " % build) npf
 
+instance Show NewPendingError where
+    show = formatToString build
+
+instance Exception NewPendingError
```

Comment:
I think it makes more sense to have automatically defined `Show` (so that it corresponds to the exception type structure, ideally should be machine readable) and use `displayException` for pretty-printing.